src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -96,4 +96,17 @@
     public static final String REQUEST_EXTRACT_CASH_ADVANCE_INDICATOR = "CASH";
     public static final String SPACING_STRING_FOR_OUTPUT = "    ";
 
+    public static class RequestExtractReport {
+        public static final String UNKNOWN_REQUEST_EXTRACT_FILENAME = "UNKNOWN_REQUEST_EXTRACT_FILENAME";
+        public static final String NO_RECORDS_WITH_VALIDATION_ERRORS_MESSAGE = "No records with validation errors.";
+        public static final String NO_VALIDATION_ERROR_MESSAGES_TO_OUTPUT = "No validation errors to output.";
+        public static final String END_OF_REPORT_MESSAGE = "End of report.";
+        public static final String PREFIX_FIRST_PART_NOT_SET_IN_CONFIGURATION_FILE = "PREFIX_FIRST_PART_NOT_SET_IN_CONFIGURATION_FILE";
+        public static final String PREFIX_SECOND_PART_NOT_SET_IN_CONFIGURATION_FILE = "PREFIX_SECOND_PART_NOT_SET_IN_CONFIGURATION_FILE";
+        public static final String REQUEST_EXTRACT_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE = "REQUEST_EXTRACT_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE";
+        public static final String REQUEST_EXTRACT_SUMMARY_REPORT_SUB_TITLE_NOT_SET_IN_CONFIGURATION_FILE = "REQUEST_EXTRACT_SUMMARY_REPORT_SUB_TITLE_NOT_SET_IN_CONFIGURATION_FILE";
+        public static final String REQUEST_EXTRACT_REPORT_CONCUR_FILE_NAME_LABEL_NOT_SET_IN_CONFIGURATION_FILE = "REQUEST_EXTRACT_REPORT_CONCUR_FILE_NAME_LABEL_NOT_SET_IN_CONFIGURATION_FILE";
+        public static final String REQUEST_EXTRACT_VALIDATION_SUB_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE = "REQUEST_EXTRACT_VALIDATION_SUB_REPORT_TITLE_NOT_SET_IN_CONFIGURATION_FILE";
```

Comment:
I am not sure we want to do that.  These are just default markers to indicate in the actual report that the actual string label is not set the way it should be. Those actual string labels conform to the standard of the other foundation reports by being defined in the spring xml file where the report writer is defined. In this case in cu-spring-concur.xml.  If anything, the actual labels in the xml should be in the cu-application.properties file but that would mean the Concur report code would not conform to the foundation code.
If we change how these defaults and actual string labels are populated for the request extract report, we will need to have a new user story created to have the same change also made for the SAE PDP/Collector Concur report.